packages/grafana-alerting/scripts/README.md

```diff
@@ -0,0 +1,5 @@
+These files are built concurrently using the `yarn run codegen` command, duplicate any of the existing config files and adapt as necessary to build the RTKQ API clients.
+
+API clients will be written to `src/grafana/api/api.<version>.gen.ts`.
```

Comment:
We were thinking about smth similar for the core API versioning, except the version would be a folder name, e.g., `src/grafana/api/api/<version>/api.gen.ts`. That way the file names don't need to be changed.

Reply:
Good idea – done!